aem-authoring-toolkit-api/src/main/java/com/exadel/aem/toolkit/api/annotations/widgets/select/Select.java

```diff
@@ -37,39 +34,103 @@
 public @interface Select {
     /**
      * Used to specify collection of {@link Option}s within this Select
+     *
      * @return Single {@code Option} annotation, or an array of Options
      */
     @IgnorePropertyMapping
     Option[] options() default {};
+
     /**
      * When set to a non-blank string, maps to the 'emptyText' attribute of this TouchUI dialog component's node.
+     *
      * @return String value
      */
     String emptyText() default "";
 
     /**
      * When set, the {@code datasource} node is appended to the JCR buildup of this component
      * and populated with values of provided {@link DataSource} annotation
+     *
      * @return {@code @DataSource} instance
      */
     @IgnorePropertyMapping
     DataSource datasource() default @DataSource;
+
     /**
+     * @return String value
      * @deprecated Use {@code datasource:resourceType} instead
      * When set to a non-blank string, allows to override {@code sling:resourceType} attribute of a {@code datasource node}
      * pointing to a ACS Commons list
-     * @return String value
      */
     @Deprecated
     @IgnorePropertyMapping
     String acsListResourceType() default "";
+
     /**
+     * @return Valid JCR path, or an empty string
      * @deprecated Use {@code datasource:path} instead
      * When set to a non-blank string, a {@code datasource} node is appended to the JCR buildup of this component
      * pointing to a ACS Commons list
-     * @return Valid JCR path, or an empty string
      */
     @Deprecated
     @IgnorePropertyMapping
     String acsListPath() default "";
+
+    /**
+     * Indicates if the user is able to select multiple selections.
+     *
+     * @return true or false
+     */
+    boolean multiple() default false;
+
+    /**
+     * Returns true to translate the options, false otherwise.
+     *
+     * @return true or false
+     */
+    boolean translateOptions() default true;
+
+    /**
+     * Returns true to sort the options based on the text, false otherwise.
```

Comment:
"If set to true, the options are sorted according to their labels' alphabetical order"